src/display.jl

```diff
@@ -0,0 +1,92 @@
+struct DisplaySimulator
+    display::Union{AbstractDisplay, Nothing}
+    render_kwargs
+    max_fps::Float64
```

Comment:
why `max_fps` rather than `fps`?

Reply:
If the simulation is slower than `max_fps` (e.g. if you are using an online solver), then it will not speed it up; it will only slow it down if it is going faster than `max_fps` so that you can see it clearly for instance